### PR TITLE
feat: generate secrets file

### LINKED ISSUE #54: separate secrets from config.toml

### PR DESCRIPTION
**Pull request by edisontim**
Closed #54

### CODE REVIEW

File: Cargo.lock
Comment (aymericdelab) [inline Cargo.lock:1]: wasn't cargo lock in gitignore ?

File: examples/starter-rpg/contracts/haiku.toml
Comment (aymericdelab) [inline examples/starter-rpg/contracts/haiku.toml:1]: signer keys still in toml ?

File: src/subcommands/build_config.rs [hunk 1]
```diff
@@ -73,9 +77,37 @@ impl BuildConfigSubcommand {
         config.haiku.db_config.vector_size = "0".to_string();
         config.haiku.db_config.number_memory_to_retrieve = "1".to_string();
         config.events = events;
+        config.haiku.llm.chat_completion_model = "Support for openai, ollama".to_string();
```
Comment (aymericdelab) [inline src/subcommands/build_config.rs:80]: why not put directly openai as default ?

**Review by aymericdelab — COMMENTED** [review]
check comments

**Review by cwastche — APPROVED** [review]

**Pull request merged**